antsibull/data/collection-enum.py

```diff
@@ -124,6 +128,39 @@ def load_all_plugins(plugin_type, basedir, coll_filter):
     return result
 
 
+def load_collection_meta_manifest(b_manifest_path, data):
+    with open(b_manifest_path, 'rb') as f:
+        meta = json.load(f)
+    data['namespace'] = meta['collection_info']['namespace']
+    data['name'] = meta['collection_info']['name']
+    data['version'] = meta['collection_info']['version']
+
+
+def load_collection_meta_galaxy(b_galaxy_path, data):
+    with open(b_galaxy_path, 'rb') as f:
+        meta = yaml.safe_load(f)
+    data['namespace'] = meta['namespace']
+    data['name'] = meta['name']
+    data['version'] = meta.get('version')
+
+
+def load_collection_meta(b_path):
+    data = {
+        'version': None,
+        'path': to_native(b_path),
+    }
+    b_manifest_path = os.path.join(b_path, to_bytes('MANIFEST.json'))
+    if os.path.exists(b_manifest_path):
+        load_collection_meta_manifest(b_manifest_path, data)
+    b_galaxy_path = os.path.join(b_path, to_bytes('galaxy.yml'))
+    b_galaxy_alt_path = os.path.join(b_path, to_bytes('galaxy.yaml'))
```

Comment:
no need to make a unicode literal and convert it into bytes using a helper. just use a byte literal
```suggestion
    b_manifest_path = os.path.join(b_path, b'MANIFEST.json')
    if os.path.exists(b_manifest_path):
        load_collection_meta_manifest(b_manifest_path, data)
    b_galaxy_path = os.path.join(b_path, b'galaxy.yml')
    b_galaxy_alt_path = os.path.join(b_path, b'galaxy.yaml')
```

Reply:
Right, I forgot that `b` already works with Python 2.7.